README.md

```diff
@@ -145,7 +145,7 @@ refer often to the following Conjur configuration variables:
 - `host_factory_token`: The Conjur host factory token, provided as a string or using the
   [Puppet file resource type](https://puppet.com/docs/puppet/latest/types/file.html).
 - `cert_file`: The file path for the PEM-encoded x509 CA certificate chain for the DAP
-  instance you are connecting to. This configuration parameter overrides `ssl_certificate`.
+  instance you are connecting to. This file is read from the Puppet server. This configuration parameter overrides `ssl_certificate`.
```

Comment:
Do you mind making the server/agent parts of these new additions bold? would make it easier to spot and not make a mistake. Also needs a linefeed.